website/docs/d/database_user.html.markdown

```diff
@@ -62,6 +62,7 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - The database user's name.
 * `roles` - List of user’s roles and the databases / collections on which the roles apply. A role allows the user to perform particular actions on the specified database. A role on the admin database can include privileges that apply to the other databases as well. See [Roles](#roles) below for more details.
 * `x509_type` - X.509 method by which the provided username is authenticated.
+* `aws_iam_type` - The new database user authenticates with AWS IAM credentials. 
```

Comment:
Edit - include what value may be here:
aws_iam_type - If set the database user authenticates with AWS IAM credentials.  Default is NONE, `USER` means user has AWS IAM user credentials, `ROLE` -  means user has credentials associated with an AWS IAM role.

Reply:
I'll add this part, thank you melissa